internal/controller/ai_gateway_route.go

```diff
@@ -59,7 +58,7 @@ func NewAIGatewayRouteController(
 	return &aiGatewayRouteController{
 		client:              client,
 		kube:                kube,
-		logger:              logger.WithName("ai-gateway-route-controller"),
+		logger:              logger.WithName("eaig-route-controller"),
```

Comment:
should be ai-eg?

Reply:
haha yeah up to you mate - feel free to fix it